assets/js/base/context/hooks/use-validation.ts

```diff
@@ -15,24 +15,25 @@ import { VALIDATION_STORE_KEY } from '@woocommerce/block-data';
 export const useValidation = (): ValidationData => {
 	const { clearValidationError, hideValidationError, setValidationErrors } =
 		useDispatch( VALIDATION_STORE_KEY );
+
+	const prefix = 'extensions-errors';
+
 	const { hasValidationErrors, getValidationError } = useSelect(
-		( select ) => {
-			const store = select( VALIDATION_STORE_KEY );
+		( mapSelect ) => {
```

Comment:
This is because we're importing `select` from `@wordpress/data` above, and due to lint rules we can't re-use the variable name (even in a different scope).